### PR TITLE
[ADD] estate: New repo and model setup T#70778

### PR DESCRIPTION
**Pull request by Gilmaciasr**
Start of the getting started section:
Chapter 1 - reading about modules structure.
https://www.odoo.com/documentation/15.0/developer/tutorials/getting_started/01_architecture.html
Chapter 2 - Setting up git and repositories, run odoo for the first time and set up the instance.
https://www.odoo.com/documentation/15.0/developer/tutorials/getting_started/02_setup.html
Chapter 3 - Create first directories with init and manifest files.
https://www.odoo.com/documentation/15.0/developer/tutorials/getting_started/03_newapp.html
Chapter 4 - estate_property model creation and attributes definitions.
https://www.odoo.com/documentation/15.0/developer/tutorials/getting_started/04_basicmodel.html
Chapter 5 -  Defining access rights and security folder. Add file to the data field in manifest.
https://www.odoo.com/documentation/15.0/developer/tutorials/getting_started/05_securityintro.html

### CODE REVIEW

**Comment by Gilmaciasr** [comment]
@JesusValdez96

**Comment by JesusValdez96** [comment]
@Gilmaciasr Please on the PR description add the link to the lessons. The same applies to the first commit.

**Review by JesusValdez96 — CHANGES_REQUESTED** [review]

**Review by JesusValdez96 — CHANGES_REQUESTED** [review]
Change the PR Title according to the guidelines: [TAG] module: Change message T#00000

File: estate/models/estate_property.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    _name = "estate.property"
+    _description = "Real state property"
+
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Datetime(default=lambda self: fields.Datetime.add(fields.Datetime.today(), days=90), copy=False)
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
```
[the remark this answers is not in the document]
Comment (JesusValdez96) [reply estate/models/estate_property.py:16]: @Gilmaciasr if you decide to keep this, add the chapter and exercise to the PR description

[the remark this answers is not in the document]
Comment (Gilmaciasr) [reply estate/models/estate_property.py]: @JesusValdez96  it's from another lesson, i'm going to change it

File: estate/models/estate_property.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        string='Orientation',
```
Comment (JesusValdez96) [inline estate/models/estate_property.py:21]: Same in other cases.

```suggestion
        string="Orientation",
```

**Review by JesusValdez96 — CHANGES_REQUESTED** [review]

**Review by JesusValdez96 — CHANGES_REQUESTED** [review]

**Comment by Gilmaciasr** [comment]
@JesusValdez96  Can you please review this again? thx

**Pull request merged**